python/lsst/sims/catUtils/mixins/VariabilityMixin.py

```diff
@@ -640,7 +641,7 @@ def applyMLTflaring(self, valid_dexes, params, expmjd,
                                "knowledge of the effective area of the LSST "
                                "mirror.")
 
-        if _MLT_LC_NPZ is None or _MLT_LC_NPZ_NAME != self._mlt_lc_file:
+        if _MLT_LC_NPZ is None or _MLT_LC_NPZ_NAME != self._mlt_lc_file or _MLT_LC_NPZ.fid is None:
```

Comment:
I'm not sure how the fid attribute gets set below and also why it would be necessary to check for it since if you load the numpy file the first two items in this list will be set.  What is different about the fileId?